python/ml4ir/base/model/optimizers/optimizer.py

```diff
@@ -0,0 +1,163 @@
+import tensorflow.keras.optimizers as tf_optimizers
+from tensorflow.keras.optimizers.schedules import ExponentialDecay
+from ml4ir.base.model.optimizers import cyclic_learning_rate
+from ml4ir.base.config.keys import OptimizerKey, LearningRateScheduleKey, CyclicLearningRateType
+
+
+def choose_optimizer(model_config, learning_rate_schedule):
```

Comment:
@mohazahran Why are we not using this https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/get

instead? I recommend changing it. Let's do it in another PR.